config/schedule.rb

```diff
@@ -39,9 +39,9 @@
 end
 
 # process the access file daily at 10:30am Eastern (the machine is in Z time, which is why this is 2pm)
-every 1.day, at: "2:30pm", roles: [:hr_cron] do
-  rake "campus_access:load", output: "/tmp/campus_access_log.log"
-end
+# every 1.day, at: "2:30pm", roles: [:hr_cron] do
```

Comment:
Should we just delete the lines?

Reply:
yes I will delete them. I wasnt sure if we would want to put it back at some point. but we can always check the history